packages/palette/src/elements/Cards/Cards.story.tsx

```diff
@@ -42,7 +42,7 @@ storiesOf("Components/Cards", module)
   .add("Large", () => {
     return (
       <LargeCard
-        image="https://www.fillmurray.com/400/300"
+        image="https://picsum.photos/seed/example/400/300"
```

Comment:
lol sorry @pvinis — we need something with a seedable random image for the stories so that Chromatic doesn't diff it in every pull request